[2023-11-17T02:17:10.219+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-16T00:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-17T02:17:10.246+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-16T00:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-17T02:17:10.268+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-17T02:17:10.277+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-16T00:00:00+00:00 [queued]>
[2023-11-17T02:17:10.277+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-17T02:17:10.291+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2023-11-16 00:00:00+00:00
[2023-11-17T02:17:10.299+0000] {standard_task_runner.py:57} INFO - Started process 14381 to run task
[2023-11-17T02:17:10.301+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2023-11-16T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpts_blsls']
[2023-11-17T02:17:10.304+0000] {standard_task_runner.py:85} INFO - Job 119: Subtask local_to_gcs_task
[2023-11-17T02:17:10.315+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-11-17T02:17:10.349+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=data_ingestion_gcs_dag/run_id=scheduled__2023-11-16T00:00:00+00:00/task_id=local_to_gcs_task permission to 509
[2023-11-17T02:17:10.352+0000] {task_command.py:416} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2023-11-16T00:00:00+00:00 [running]> on host 2952db3161f1
[2023-11-17T02:17:10.430+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-16T00:00:00+00:00'
[2023-11-17T02:17:11.935+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-17T02:17:11.951+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20231116T000000, start_date=20231117T021710, end_date=20231117T021711
[2023-11-17T02:17:12.004+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-17T02:17:12.035+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
